Update cpp.md (#3608)
Codecademy/docs: content/cpp/cpp.md

**C++** is a very powerful programming language for performance-critical applications that rely on speed and efficient memory management. It’s used in a wide range of industries including software and game development, robotics, microcontrollers, VR/AR, and scientific computing.

The C++ language was created by [Bjarne Stroustrup](https://www.codecademy.com/resources/docs/general/historical-technical-figures/bjarne-stroustrup) at Bell Laboratories in 1979. As the name implies, C++ was derived from the [C](https://www.codecademy.com/resources/docs/c) language; Bjarne’s goal was to add [object-oriented programming](https://www.codecademy.com/resources/docs/general/programming-paradigms/object-oriented-programming) into C, a language well-respected for its portability and low-level functionality.

> There are a lot of versions of the C++ programming language. These versions of the language are implementations of compilers based on specifications constructed by the ISO C++ community, the community that oversees the development of the language. The following are the versions of the language −

Reviewer: ```suggestion

There are many of versions of the C++ programming language. These versions of the language are implementations of compilers based on specifications constructed by the ISO C++ community, the community that oversees the development of the language. The following are versions of the language:
```
Needs a blank line between this and the preceding paragraph.

> C++98 (ISO/IEC 14882:1998) is the first edition.
> C++03 (ISO/IEC 14882:2003) is the second edition.
> C++11 is the third edition.
> C++14 is the fourth edition.
> C++17 is the fifth edition.

Reviewer: ```suggestion
- C++98 (ISO/IEC 14882:1998) is the first edition.
- C++03 (ISO/IEC 14882:2003) is the second edition.
- C++11 is the third edition.
- C++14 is the fourth edition.
- C++17 is the fifth edition.

```
Needs to be a bulleted list and have a blank line following it.

> All of these versions have a lot of differences, mainly additions to the standard library and expansion of APIs. These standards also have differences in APIs due to deperaction.

Reviewer: ```suggestion
These versions have many of differences, mainly additions to the standard library and expansion of APIs. These standards also have differences in APIs due to depreciation.
```
